.github/workflows/publish.yaml

```diff
@@ -0,0 +1,71 @@
+name: '🚀 Build and Publish jellyfin-ffmpeg'
+
+on:
+  release:
+    types:
+      - released
+
+jobs:
+  build_publish_debian:
+    uses: jellyfin/jellyfin-ffmpeg/.github/workflows/_meta.yaml@master
+    with:
+      distro: 'debian'
+      codenames: '["buster", "bullseye"]'
+      architectures: '["amd64", "arm64", "armhf"]'
+      release: true
+
+  build_publish_ubuntu:
+    uses: jellyfin/jellyfin-ffmpeg/.github/workflows/_meta.yaml@master
+    with:
+      distro: 'ubuntu'
+      codenames: '["focal", "bionic", "impish", "hirsute"]'
+      architectures: '["amd64", "arm64", "armhf"]'
+      release: true
+
+  build_publish_windows:
+    uses: jellyfin/jellyfin-ffmpeg/.github/workflows/_meta.yaml@master
+    with:
+      distro: 'windows'
+      codenames: '["Windows 10"]'
+      architectures: '["win64"]'
+      release: true
+
+  maintain_repository:
+    name: Maintain Repository
+    if: ${{ !github.event.release.prerelease && inputs.release }}
+    runs-on: ubuntu-latest
+    needs:
+      - build_publish_debian
+      - build_publish_ubuntu
+      - build_publish_windows
+    strategy:
+      fail-fast: false
+      matrix:
+        distro:
+          - debian
+          - ubuntu
+          - windows
+    steps:
+      - name: Sync mirrors and update symlinks
+        uses: appleboy/ssh-action@v0.1.4
+        with:
+          host: ${{ secrets.deploy-host }}
+          username: ${{ secrets.deploy-user }}
+          key: ${{ secrets.deploy-key }}
+          script_stop: true
+          script: |-
+            if [ "${matrix.distro}" != "windows" ]; then
+              sudo find /srv/repository/releases/server/${matrix.distro}/ -type l -name "jellyfin-ffmpeg_*" -exec rm {} \;
+              reprepro -b /srv/repository/${matrix.distro} deleteunreferenced
+              reprepro -b /srv/repository/${matrix.distro} export
+              find "/srv/repository/${matrix.distro}" -name "*.deb" | while read file; do
```

Comment:
This should be a similar consruct to the find above, to ensure it's looking in the right place, or even more specific right down into the exact folder where the upload puts the files.

I'd actually much prefer this upload into a temporary location (i.e. `/srv/repository/incoming/ffmpeg`) then import those into `reprepro`, then finally move them to the final place and update symlinks. It seems safer that way.

Reply:
I think I got it right this time.